### ISSUE TITLE
Remove Jquery usages

### ISSUE THREAD

**Issue by RomainOdeval** [RomainOdeval/batflat#8]
It's time to leave, JQuery !

**Comment by WebCrew** [comment]
Great job and a **big thank you for the upgrade**. 

I hope you keep up with it for a little longer, that would be awesome. 

It is such a nice little FF-CMS would be a pain if it is left to die. 

If I would be a way better with PHP I would even try to help, but it is better I dont do it - **believe me** :)

**Kind regards**, Andreas :)

**Comment by RomainOdeval** [comment]
> 

Hello WebCrew,

Thank you for your appreciation !
To be fair, it's complicated for me to do "extra work" now so i didn't work on this project since ... a lot !

BUT i check it and i think i removed last time all jQuery occurences.
I think the issue will be now more "Find a way to insert js properly, like with npm", somethin like that.

If you have needs, don't hesitate to create issues.
I can't guarantee i will work on it but, how knows, if it's easy, maybe i will do it 👍

**Comment by WebCrew** [comment]
Thanks for the offer Romain. Oh and take your time at the end its only a hobby. 

I could be wrong, but I think the dependencies can also be updated quickly with Composer, but npm is certainly a solution too. 

If I remember correctly I found someone on GitHub today, who made a fork from your repo and he did also some updates to the CMS too. Maybe he already solved the problem? Unfortunately I can't remember his name, but I think you can find out if you look up the people who cloned your repo. 

Kind regards, Andreas ;)

**Comment by RomainOdeval** [comment]
> Thanks for the offer Romain. Oh and take your time at the end its only a hobby.
> 
> I could be wrong, but I think the dependencies can also be updated quickly with Composer, but npm is certainly a solution too.
> 
> If I remember correctly I found someone on GitHub today, who made a fork from your repo and he did also some updates to the CMS too. Maybe he already solved the problem? Unfortunately I can't remember his name, but I think you can find out if you look up the people who cloned your repo.
> 
> Kind regards, Andreas ;)

oh, if you talk about [ylorant](https://github.com/ylorant) / [sta-website](https://github.com/ylorant/sta-website) , it's my friend and it's a fork for the association i initially rework this cms. 
I saw some fixes, il will take them.

I close this issue since all JQuery is removed.

**Comment by WebCrew** [comment]
Yes it was the sta-website I think he also updated the upgrade.php file of the CMS and the query builder etc. Maybe he has also an idea because of the future updates of the JS dependencies.

Wish you a great day Romain. :)

**Comment by RomainOdeval** [comment]
At this point, here a to-do list to finish this issue : 

- [ ] Resolve warning "choices.min.js:2 Trying to initialise Choices on element already initialised {element: select.form-control.choices__input}" on "Blog/ add or edit entry". Check file "inc/modules/blog/js/admin/blog.js", near line 197
- [ ] Remove "BootboxJS" with a non-Jquery alternative
- [ ] Redo "Notifications" logic in "themes/admin/js/scripts.js"
- [ ] Move all remaining JQuery code from html files into js files and transform it into VanillaJS (biggest part)
- [ ] Transform remaining JQuery code from js files into VanillaJS (second biggest part).
- [ ] Remove JQuery 3.7.0 call
